### DISCUSSION TITLE
en/blog/en/indie-hacker-tech-stack-2024/

### DISCUSSION THREAD

**Discussion by giscus** [guangzhengli/blog-comments#18] [Announcements]
# en/blog/en/indie-hacker-tech-stack-2024/

Ladder@The purpose of writing this article is to celebrate reaching 10K followers on X/Twitter 🎉. Suddenly, I felt the urge to share. In today’s world, where the choice of tech stacks and cloud service providers is so rich and diverse, what tech stack should we choose if we need to develop independently?
Why Independent Development? Why do I limit my technology choices to independent development? Because if we are developing for a company or a client, I feel that the room for technological choices is limited, and the fun is also reduced.

https://guangzhengli.com/en/blog/en/indie-hacker-tech-stack-2024/

**Comment by hhamja** [comment]
hi :)
Thanks for posting
When talking about cloudflare, you mentioned that you gave it 5 points. Is there a link to that article?
